pywbem/tupleparse.py

```diff
@@ -949,7 +1024,7 @@ def parse_instance(tup_tree):
             %ClassName;>
     """
 
-    check_node(tup_tree, 'INSTANCE', ['CLASSNAME'],
+    check_node(tup_tree, 'INSTANCE', ['CLASSNAME'], [],
```

Comment:
Karl. Check why the empty list entry

Reply:
This is parameter was missing and adding it is a fix.
It specifies the allowed optional attributes, so before the change, the list of child elements was in that argument position and was interpreted as optional attributes. And the argument position for the list of child elements was unspecified, so it defaulted to None which means any child element was allowed.